index.js

```diff
@@ -0,0 +1,152 @@
+$(document).ready(() => {
+  listTrips();
+  loadTripDetails();
+});
```

Comment:
Traditionally `$(document).ready` would go at the bottom of the script. There's not really a good reason for this, it's just a style thing.